react/features/toolbox/defaultToolbarButtons.js

```diff
@@ -153,12 +154,32 @@ export default {
         id: 'toolbar_button_dial_out',
         onClick() {
             JitsiMeetJS.analytics.sendEvent('toolbar.sip.clicked');
-            APP.store.dispatch(openDialOutDialog());
+
+            return openDialOutDialog();
         },
         tooltipKey: 'dialOut.dialOut'
     },
 
     /**
+     * The descriptor of the device selection toolbar button.
+     */
+    deviceselection: {
```

Comment:
Let's just rename it to something that suggests it's filmstrip-only button. Something like: fo-deviceselection. We can find a better way to do this in the future.